### ISSUE TITLE
Add alt-text for ml-ops-cycle fig on main page

### ISSUE THREAD

**Issue by batpigandme** [rstudio/vetiver.rstudio.com#7]
The ML-Ops Cycle figure currently has no alt text.

Full description of issue generated by axe here: https://axe.deque.com/issues/aa5f97db-69dd-4ed6-9629-a3d70d7e94c8

Relevant line in the docs is here:
https://github.com/juliasilge/vetiver.dev/blob/4424dadcf46e5a1cf9bf40784ebe4dc6b31c3381/index.qmd#L14

Solution: Add a space in after `fig-align="center"` in the curly braces followed by `fig-alt="Whatever the alt text should be."`

**Comment by juliasilge** [comment]
Closed in 546630b8ae7545d80e708f93f58f9d956fed3e18
